docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc

```diff
@@ -73,6 +74,13 @@ The Transactions report is a Technology Preview feature only. Technology Preview
 +
 [NOTE]
 ====
+Note that the automatically attached tags are displayed only _after_ you run the analysis.
+
+Instead of enabling automated tagging or in addition to it, you can attach tags to the application manually.
```

Comment:
Suggestion. "You can attach tags to the application manually, instead of automated tagging or in addition to it."

Reply:
Done